ci/release.sh

```diff
@@ -1,11 +1,10 @@
 # Generate the node-gyp formatted filename from the node environment
 FILENAME=$(
   node -e "
-    var p = process, v = p.versions, pkg = require('./package.json');
-    console.log([pkg.name, 'v$CANVAS_PREBUILT_VERSION', 'node-v' + v.modules, p.platform, p.arch].join('-'));
+    var p = process, v = p.versions, pkg = require('./package.json'), libc = require('detect-libc').family;
+    console.log([pkg.name, 'v$CANVAS_PREBUILT_VERSION', 'node-v' + v.modules, p.platform, libc, p.arch].join('-'));
```

Comment:
Thanks. Doesn't `detect-libc` need to be added to `package.json` though? Also ignore the failed builds, it's trying to do a release

Reply:
You're right. I'll add it.

Reply:
Done.